lastuser_core/models/user.py

```diff
@@ -119,7 +119,7 @@ def username(self, value):
         elif self.is_valid_username(value):
             self._username = value
 
-    # Alias name to username
+    # Alias username to name
```

Comment:
This uses `alias` in the verb form, which implies "name" is an alias for "username". The old comment was correct. https://en.wiktionary.org/wiki/alias#Verb